data/src/main/scala/cats/data/Or.scala

```diff
@@ -381,6 +374,10 @@ sealed abstract class OrInstances extends OrInstances0 {
       def empty =
         ROr(Monoid[B].empty)
     }
+
+  implicit def OrdShow[A: Show, B: Show]: Show[A Or B] =
```

Comment:
I should probably add a note about why that is my preference. I think it just avoids the 2 extra function calls to Show.apply[A], so I assume it is just slightly better from a performance perspective.